lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -39,7 +39,7 @@ Markdown = "*"
 # Optional requirement for markdown export.
 # Weasyprint = "==0.42"
 PyYAML = "*"
-SQLAlchemy = "*"
+SQLAlchemy = ">=1.3.22, <1.4.0"  # https://github.com/kvesteri/sqlalchemy-utils/issues/474
```

Comment:
This should have been 1.3.20 (the version we are currently pinning in `requirements.txt` for 21.01 , but not a big deal, I don't think we are going to use pipenv to update deps in the release branch.